frontend/app/routes/$lang+/_public+/apply+/$id+/adult-child/review-child-information.tsx

```diff
@@ -262,71 +214,87 @@ export default function ReviewInformation() {
-                    {t('apply-adult-child:review-child-information.dental-benefit-change')}
-                  </InlineLink>
-                </p>
-              </DescriptionListItem>
-            </dl>
+        {children.map((childData, index) => (
+          <div key={childData.id} className="space-y-10">
+            <h2 className="text-4xl font-semibold">{childData.information?.firstName}</h2>
+            <div>
+              <h2 className="text-2xl font-semibold">{t('apply-adult-child:review-child-information.page-sub-title', { child: childData.information?.firstName })}</h2>
+              <dl className="mt-6 divide-y border-y">
+                <DescriptionListItem term={t('apply-adult-child:review-child-information.full-name-title')}>
+                  {`${childData.information?.firstName} ${childData.information?.lastName}`}
+                  <p className="mt-4">
+                    <InlineLink id="change-full-name" routeId="$lang+/_public+/apply+/$id+/adult/applicant-information" params={params}>
+                      {t('apply-adult-child:review-child-information.full-name-change')}
+                    </InlineLink>
+                  </p>
+                </DescriptionListItem>
+                <DescriptionListItem term={t('apply-adult-child:review-child-information.dob-title')}>
+                  {childData.information?.dateOfBirth}
+                  <p className="mt-4">
+                    <InlineLink id="change-date-of-birth" routeId="$lang+/_public+/apply+/$id+/adult/date-of-birth" params={params}>
+                      {t('apply-adult-child:review-child-information.dob-change')}
+                    </InlineLink>
+                  </p>
+                </DescriptionListItem>
+                <DescriptionListItem term={t('apply-adult-child:review-child-information.sin-title')}>
+                  {childData.information?.socialInsuranceNumber ? formatSin(childData.information.socialInsuranceNumber) : ''}
+                  <p className="mt-4">
+                    <InlineLink id="change-sin" routeId="$lang+/_public+/apply+/$id+/adult/applicant-information" params={params}>
+                      {t('apply-adult-child:review-child-information.sin-change')}
+                    </InlineLink>
+                  </p>
+                </DescriptionListItem>
+                <DescriptionListItem term={t('apply-adult-child:review-child-information.is-parent')}>
+                  {childData.information?.isParent ? t('apply-adult-child:review-child-information.yes') : t('apply-adult-child:review-child-information.no')}
+                </DescriptionListItem>
+              </dl>
+            </div>
+            <div>
+              <h2 className="mt-8 text-2xl font-semibold">{t('apply-adult-child:review-child-information.dental-title', { child: childData.information?.firstName })}</h2>
```

Comment:
Use `h3` instead of `h2` with the following classes `mb-6 font-lato text-2xl font-semibold`.

Reply:
Changed to h3
Updated className